mspray/apps/main/tests/views/test_performance.py

```diff
@@ -34,3 +35,43 @@ def test_team_leader_performance(self):
         response = view(request, slug=sprayarea.id)
         self.assertEqual(response.status_code, 200)
         self.assertTrue('Supervisor' in str(response.render().content))
+
+    def test_mda_spray_operator_summary_view(self):
+        """Test MDASprayOperatorSummaryView."""
+        data_setup()
+        sprayarea = Location.objects.get(name='Mtendere')
+
+        factory = RequestFactory()
+        request = factory.get("spray-operators/2/summary")
+
+        view = MDASprayOperatorSummaryView.as_view()
+        response = view(request, rhc_id=sprayarea.id)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            response.context_data['rhc_name'], "Mtendere")
+
+        queryset = SprayOperator.objects.raw(
+            MDA_SOP_PERFORMANCE_SQL, [sprayarea.id]
+        )
+        serializer = SprayOperatorPerformanceReportSerializer(
+            queryset, many=True
+        )
+        result = {
```

Comment:
Unfortunately with no actual calculations this test may not be sufficient to test that our calculations actually work.

Please consider using `mspray.apps.main.tests.utils.load_spray_data` to load spray data so that the result actually has some numbers.